coordinator/src/models/fhir/fhir-resources.ts

```diff
@@ -45,19 +51,23 @@ interface BaseMedicationRequest extends Resource {
   authoredOn: string
   requester: Reference<PractitionerRole>
   groupIdentifier: MedicationRequestGroupIdentifier
-  dispenseRequest: MedicationRequestDispenseRequest
+  dispenseRequest?: MedicationRequestDispenseRequest
   substitution?: {
     allowedBoolean: false
   }
 }
 
+//TODO - at what point do we just use Extension instead of a union type? What benefit is this providing?
```

Comment:
The only benefit is excluding extension types which we know can never be included in a medication. If there are none, then it makes sense to move to using Extension instead

Reply:
We're not there yet - there's like 50 types of Extension in FHIR